app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -1100,19 +1100,15 @@ export default class SampleViewV2 extends React.Component {
           <div className={cs.sampleViewHeader}>
             <SampleViewHeader
               backgroundId={selectedOptions.background}
+              deletable={sample ? sample.deletable : false}
               editable={sample ? sample.editable : false}
               onDetailsClick={this.toggleSampleDetailsSidebar}
               onPipelineVersionChange={this.handlePipelineVersionSelect}
               pipelineRun={pipelineRun}
+              pipeilneRunStatus={reportMetadata.pipeilneRunStatus}
```

Comment:
Typo! Should be `pipelineRunStatus={reportMetadata.pipelineRunStatus}`

Reply:
Good catch... actually I end up not using this variable... removed